.travis.yml

```diff
@@ -11,9 +11,12 @@ script:
 - yarn lint
 - yarn test
 - yarn build:css:all
+- yarn build:doc:react
+- yarn screenshots
+- yarn argos --token $ARGOS_TOKEN --branch $TRAVIS_BRANCH --commit $TRAVIS_COMMIT
```

Comment:
So you have configured on the `argos` side that `master` is the "default" and like that `argos` will check the diff between `master` and `$TRAVIS_BRANCH` ? 

Can't wait to have a PDC on that subject :)

Reply:
Argos detects that master is the default by looking at the base branch of the repository.